documentation/staging/content/faq/external-clients.md

```diff
@@ -366,6 +381,129 @@ To enable an 'unknown host' source WebLogic Server to initiate EJB, JMS, or JTA
     * For operator hosted WebLogic Servers, you can set this property by including `-Dweblogic.rjvm.allowUnknownHost=true` in the `JAVA_OPTIONS` [Domain environment variable]({{< relref "/userguide/managing-domains/domain-resource#jvm-memory-and-java-option-environment-variables" >}}) defined in the domain resource's `spec.serverPod.env` attribute.
   * Also apply patch 30656708 on each target WebLogic Server for versions 12.2.1.4 (PS4) or earlier.
 
+#### Configuring WebLogic Server Affinity Load Balancing Algorithm
+
+When providing external clients access to EJB and JMS resources using load balancer tunneling, it is recommended to configure a server affinity load balancing algorithm for the cluster in which the resources are targeted.  Using a server affinity based algorithm turns off load balancing between external Java clients and server instances causing method calls to 'stick' to a server instance to which the external client is communicating with.  For more information see [Load Balancing for EJBs and RMI Objects](https://docs.oracle.com/en/middleware/fusion-middleware/weblogic-server/12.2.1.4/clust/load_balancing.html#GUID-2470EEE9-F6F9-44EF-BA54-671728E93DE6) in _Load Balancing in a Cluster_.
+
+Here are steps for enabling server affinity in a cluster, named 'cluster-1', for a domain source type __Domain in PV__:
+
+```
+How to implement Server Affinity in Cluster: 
+=========================================== 
+1. Go to  admin console 
+2. Expand the Clusters link on the left panel 
+3. Click on 'cluster-1' link 
+    i.   look on the right panel. 
+    ii.  look for Default Load Algorithm property 
+    iii. change the value to "RoundRobinAffinity" instead of "RoundRobin". 
+4. Restart the domain/weblogic server instances. 
```

Comment:
In WKO environment all the changes to config.xml is lost if we do not use the Domain-on-PV model. So this is not a suggested way to modify the cluster attribute.

Reply:
I think you clearly document that this is for teh domain-on-PV model, which mitigates Pani's concern, but I also think it isn't the preferred way to make a change even for domain-on-PV.  The WLST offline & WDT snippets are sufficient IMO.  The WL console instructions can be removed. 

Ditto for the other locations Pani made the same comment...